### PR TITLE
[ADD] delivery_carrier_label_ups

### PR DESCRIPTION
**Pull request by hbrunn**
This adds carrier and label integration for [UPS](ups.com)

### CODE REVIEW

**Review by StefanRijnhart — APPROVED** [review]
Approving after passing review on the internal PR on our customer project.

File: delivery_carrier_label_ups/models/product_packaging.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
```
Comment (rousseldenis) [inline delivery_carrier_label_ups/models/product_packaging.py:7]: Do you really use product packaging this way ? Why don't you use stock.quant.package ?

Comment (rousseldenis) [reply delivery_carrier_label_ups/models/product_packaging.py:7]: Ok, understood, you use packaging_id on stock.quant https://github.com/odoo/odoo/blob/12.0/addons/stock/models/stock_quant.py#L354

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**